src/gei/Handlers/MigrateOrgCommandHandler.cs

```diff
@@ -50,13 +50,21 @@ public async Task Handle(MigrateOrgCommandArgs args)
             return;
         }
 
-        var (migrationState, _, _, failureReason) = await _githubApi.GetOrganizationMigration(migrationId);
+        var (migrationState, _, _, failureReason, completedRepositoriesCount, totalRepositoriesCount) = await _githubApi.GetOrganizationMigration(migrationId);
 
         while (OrganizationMigrationStatus.IsPending(migrationState))
         {
-            _log.LogInformation($"Migration in progress (ID: {migrationId}). State: {migrationState}. Waiting 10 seconds...");
+            if (OrganizationMigrationStatus.IsRepoMigration(migrationState))
+            {
+                var remainingRepositoriesCount = (int)totalRepositoriesCount - (int)completedRepositoriesCount;
+                _log.LogInformation($"Migration in progress (ID: {migrationId}). State: {migrationState}. {remainingRepositoriesCount}/{totalRepositoriesCount} repo(s) migrated. Waiting 10 seconds...");
```

Comment:
this logic or variable names is confusing. shouldn't the message be {completed}/{total} not {remaining}/{total} ?

Reply:
You're totally right. My brain wasn't working when I wrote this PR. 

The API returns remaining and total and we want to present it as `completed/remaining`. I'll fix this and request a re-review.

Reply:
I just misnamed the variable returned from the API in this method and then went from there 😂